src/MongoDB.Driver.Core/Core/Configuration/ConnectionString.cs

```diff
@@ -747,6 +747,23 @@ private string GetHostNameForDns()
             return host;
         }
 
+#if !NETSTANDARD1_5
+        private static bool AllPercentEncodingsAreValid(string value)
+        {
+            for (var i = 0; i < value.Length; i++)
+            {
+                //Uri.IsHexEncoding is not in .NETStandard 1.5, which was my default on Rider
+                //also appears to not compile if I use .NETFramework
```

Comment:
We like to put one space after "//" before the start of the text.

Reply:
That's unfortunate about .NET Standard 1.5.

Do you think it's worth having two different implementations? Or should we just use regular expressions on both platforms?

Reply:
I'm fine with simplifying things and sticking with the regular expression approach.